components/org.wso2.carbon.identity.api.idpmgt.endpoint/src/main/resources/identity-provider-management.yaml

```diff
@@ -992,6 +995,25 @@ parameters:
     default: 0
     type: integer
 
+# Used for filtering idps:
+  filter:
+    name: filter
+    in: query
+    required: false
+    description: |
+      Condition to filter the retrival of idps.
+      Only support filtering with idp name.
+    type: string
+
+# Used for sorting idps:
+  sort:
```

Comment:
Started the comment to propose 'orderBy' instead of 'sort' as the keyword.

When checking some real usages,
* Github use 'sort': https://help.github.com/en/articles/sorting-search-results
* SCIM uses 'sortBy': https://tools.ietf.org/html/rfc7644#section-3.4.2.3

And WSO2 REST API Guideline also use 'sort' as the example query param name to specify the order.
https://wso2.com/whitepapers/wso2-rest-apis-design-guidelines/#102

So decided to stick with the 'sort' :)